Explorer/Assets/Scripts/SceneRuntime/Apis/Modules/RestrictedActionsApi/IRestrictedActionsAPI.cs

```diff
@@ -8,11 +7,15 @@ namespace SceneRuntime.Apis.Modules.RestrictedActionsApi
     public interface IRestrictedActionsAPI
     {
         bool TryOpenExternalUrl(string url);
+
         void TryMovePlayerTo(Vector3 newRelativePosition, Vector3? cameraTarget);
+
         void TryTeleportTo(Vector2Int newCoords);
+
         bool TryChangeRealm(string message, string realm);
-        void TryTriggerEmote(string predefinedEmote);
```

Comment:
Why this got deleted?

Reply:
I got no clue, probably during a mega merge :S restored it now
I remember I had already restored it before, not sure what happened with these files...